### PR TITLE
feat: translate module, add materials

### PR DESCRIPTION
**Pull request by Santerna**
#### 🤔 This is a ...

- [ ] 🌟 New task
- [ ] 🌐 New module
- [ ] ⚙️ Update to an existing task
- [x] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [x] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description
- [x] Module description translated
- [x] Module topics and materials list expanded
- [x] Module materials revised and updated

#### Additional Information

Issue [1464](https://github.com/rolling-scopes-school/tasks/issues/1464)

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: stage2/modules/testing/README.md [hunk 1]
```diff
@@ -1,36 +1,49 @@
...
 
-- Overview
-- Testing Types
+In this module, the fundamentals of testing in JavaScript covered, including its importance, different types of testing, popular frameworks and libraries, methodologies like FIRST, TDD and BDD, and best practices. Testing is an integral part of the software development process, and mastering it can greatly improve the quality and reliability of application code.
+
+## Learning Objectives 🎯
+
+Students will:
+
+- Study the basics of testing and see the importance of testing in JavaScript development
```
Comment (haslie22) [inline stage2/modules/testing/README.md:11]: _Study the basics of testing and **understand** the importance..._ sounds a little bit better, in my opinion 🤔

File: stage2/modules/testing/README.md [hunk 2]
```diff
@@ -1,36 +1,49 @@
...
   - Integration Testing
   - E2E
   - Security Testing
   - Perforamance Testing
-- Test Pyramid
-- Testing approaches
+- Understand the concept of Test Pyramid
+- Differentiate between testing approaches like:
   - FIRST
   - TDD и BDD
```
Comment (haslie22) [inline stage2/modules/testing/README.md:21]: TDD **and** BDD

File: stage2/modules/testing/README.md [hunk 3]
```diff
@@ -1,36 +1,49 @@
...
 
-## Теория
+- https://testsigma.com/blog/fundamentals-of-software-testing/
+- https://www.testim.io/blog/software-testing-basics/
+- https://circleci.com/blog/test-driven-development-tdd/
+- https://blog.logrocket.com/javascript-testing-best-practices/
+- https://wanago.io/2018/08/27/testing-javascript-tutorial-types-of-tests-of-unit-testing-with-jest/
+- https://github.com/goldbergyoni/javascript-testing-best-practices
+- https://www.testim.io/blog/javascript-integration-testing-painlessly/
+- https://testsigma.com/blog/end-to-end-testing-frameworks/
```
Comment (haslie22) [inline stage2/modules/testing/README.md:39]: Please align links formatting with the [module template](https://github.com/rolling-scopes-school/tasks/blob/master/stage1/modules/module_template/module_template.md)

File: stage2/modules/testing/README.md [hunk 4]
```diff
@@ -1,36 +1,49 @@
...
+- Complete the "Testing" test in the RS-App > Auto Test.
 
-- Вам необходимо пройти тест "Testing" в RS APP > Auto Test
+## Additional Resources 📘
 
-## Дополнительные материалы
+Expand your knowledge with these additional materials:
 
-- https://www.youtube.com/watch?v=xrS60rkoG3w&t
+- https://www.youtube.com/watch?v=XnsV8YACAHQ
```
Comment (haslie22) [inline stage2/modules/testing/README.md:49]: Please align links formatting with the [module template](https://github.com/rolling-scopes-school/tasks/blob/master/stage1/modules/module_template/module_template.md)

File: stage2/modules/testing/README.md [hunk 5]
```diff
@@ -1,36 +1,49 @@
...
 
-- https://doka.guide/tools/how-to-test-and-why/
-- https://doka.guide/tools/tdd/
-- https://doka.guide/tools/how-to-simplify-tests/
-- https://doka.guide/tools/testing-and-fake-objects/
-- https://doka.guide/tools/integration-and-system-testing/
+## Practice 💻
 
-## Практика
+- Complete the "Testing" test in the RS-App > Auto Test.
```
Comment (haslie22) [inline stage2/modules/testing/README.md:43]: Complete the "**[St2]** Testing" test in the RS-App > Auto Test.

Comment (valerydluski) [reply stage2/modules/testing/README.md:43]: also please add a link to the task [RSS Testing](./tasks/rss-testing/)

**Review by valerydluski — COMMENTED** [review]
Thank you

File: stage2/modules/testing/README.md [hunk 6]
```diff
@@ -1,36 +1,49 @@
...
 
-- Вам необходимо пройти тест "Testing" в RS APP > Auto Test
+## Additional Resources 📘
 
-## Дополнительные материалы
+Expand your knowledge with these additional materials:
 
-- https://www.youtube.com/watch?v=xrS60rkoG3w&t
```
Comment (valerydluski) [inline stage2/modules/testing/README.md:36]: This lecture I would also keep
and add this https://youtu.be/qFSrImO04X8

File: stage2/modules/testing/README.md [hunk 7]
```diff
@@ -1,36 +1,49 @@
...
 
-## Теория
+- https://testsigma.com/blog/fundamentals-of-software-testing/
+- https://www.testim.io/blog/software-testing-basics/
+- https://circleci.com/blog/test-driven-development-tdd/
+- https://blog.logrocket.com/javascript-testing-best-practices/
+- https://wanago.io/2018/08/27/testing-javascript-tutorial-types-of-tests-of-unit-testing-with-jest/
+- https://github.com/goldbergyoni/javascript-testing-best-practices
+- https://www.testim.io/blog/javascript-integration-testing-painlessly/
+- https://testsigma.com/blog/end-to-end-testing-frameworks/
```
Comment (valerydluski) [inline stage2/modules/testing/README.md:40]: please add this article
https://www.atlassian.com/continuous-delivery/software-testing/types-of-software-testing

Comment (Santerna) [reply stage2/modules/testing/README.md:41]: Article added

File: stage2/modules/testing/README.md (hunk 6 above)
Comment (Santerna) [reply stage2/modules/testing/README.md:36]: Video added

File: stage2/modules/testing/README.md (hunk 5 above)
Comment (Santerna) [reply stage2/modules/testing/README.md]: Fixed

File: stage2/modules/testing/README.md (hunk 4 above)
Comment (Santerna) [reply stage2/modules/testing/README.md]: Formatted

File: stage2/modules/testing/README.md (hunk 3 above)
Comment (Santerna) [reply stage2/modules/testing/README.md]: Formatted

File: stage2/modules/testing/README.md (hunk 2 above)
Comment (Santerna) [reply stage2/modules/testing/README.md]: Fixed:)

File: stage2/modules/testing/README.md (hunk 1 above)
Comment (Santerna) [reply stage2/modules/testing/README.md]: Fixed

File: stage2/modules/testing/README.md [hunk 8]
```diff
@@ -1,36 +1,49 @@
-# [Stage#2.](../../) Введение в тестирование
+# [Stage#2.](../../) Testing
 
-## В данном модуле студентам необходимо изучить:
+## Module Overview 📚
 
-- Overview
-- Testing Types
+In this module, the fundamentals of testing in JavaScript covered, including its importance, different types of testing, popular frameworks and libraries, methodologies like FIRST, TDD and BDD, and best practices. Testing is an integral part of the software development process, and mastering it can greatly improve the quality and reliability of application code.
```
[the remark this answers is not in the document]
Comment (Santerna) [reply stage2/modules/testing/README.md]: Fixed

File: stage2/modules/testing/README.md [hunk 9]
```diff
@@ -1,36 +1,52 @@
...
+- **2 hours**
 
-## Приблизительное время прохождения модуля
+## Theory 📖
 
-2 часа
+Students are encouraged to study the following resources:
 
-## Теория
+- [Fundamentals of Software Tseting](https://testsigma.com/blog/fundamentals-of-software-testing/) - [20min]
```
Comment (haslie22) [inline stage2/modules/testing/README.md:32]:
```suggestion
- [Fundamentals of Software Testing](https://testsigma.com/blog/fundamentals-of-software-testing/) - [20min]
```

File: stage2/modules/testing/README.md [hunk 10]
```diff
@@ -1,36 +1,52 @@
...
 
-- https://doka.guide/tools/how-to-test-and-why/
-- https://doka.guide/tools/tdd/
-- https://doka.guide/tools/how-to-simplify-tests/
-- https://doka.guide/tools/testing-and-fake-objects/
-- https://doka.guide/tools/integration-and-system-testing/
+## Practice 💻
 
-## Практика
+- Complete the ["[St2] Testing"](https://github.com/rolling-scopes-school/tasks/rss-testing/) test in the RS-App > Auto Test.
```
Comment (haslie22) [inline stage2/modules/testing/README.md:44]: Maybe we should remove the link to the rss-testing task? This task won't be available for the students in the current course.

File: stage2/modules/testing/README.md [hunk 11]
```diff
@@ -1,36 +1,52 @@
...
-- Вам необходимо пройти тест "Testing" в RS APP > Auto Test
+## Additional Resources 📘
 
-## Дополнительные материалы
+Expand your knowledge with these additional materials:
 
-- https://www.youtube.com/watch?v=xrS60rkoG3w&t
+- [Fail Fast approach in Frontend Development](https://www.youtube.com/watch?v=XnsV8YACAHQ) - [50min]
+- [Testing](https://www.youtube.com/watch?v=xrS60rkoG3w&t) - [52min]
+- [Testing live coding](https://youtu.be/qFSrImO04X8) - [1h37min]
```
Comment (haslie22) [inline stage2/modules/testing/README.md:52]:
```suggestion
- [Testing live coding](https://youtu.be/qFSrImO04X8) - [1h 37min]
```

**Review by haslie22 — COMMENTED** [review]
Thank you! 👍 
Left a few comments with minor fixes

**Review by valerydluski — APPROVED** [review]

**Pull request merged**